service/storage_service.cc

```diff
@@ -614,8 +614,12 @@ void storage_service::bootstrap() {
     if (!_db.local().is_replacing()) {
         // Wait until we know tokens of existing node before announcing join status.
         _gossiper.wait_for_range_setup().get();
-
-        if (get_token_metadata_ptr()->count_normal_token_owners() == 0) {
+        int retry = 0;
+        while (get_token_metadata_ptr()->count_normal_token_owners() == 0) {
+            if (retry++ < 500) {
```

Comment:
I do not want to delay the abort too much. Because normally without the tricks to boot fast for testing, we should abort immediately. We do not want the user to see the error 10 minutes later.

Reply:
Ok, I think I understand now. Before this change it didn't wait at all, so it's not like suddenly we should add 10 minutes of wait. Ok. Good.